roc/receiver_test.go

```diff
@@ -9,12 +9,17 @@ import (
 
 func TestReceiver_Open(t *testing.T) {
 	tests := []struct {
-		name    string
-		config  ReceiverConfig
-		wantErr error
+		name        string
+		contextFunc func() *Context
+		config      ReceiverConfig
+		wantErr     error
 	}{
 		{
 			name: "ok",
+			contextFunc: func() *Context {
+				ctx, _ := OpenContext(ContextConfig{})
```

Comment:
I'd suggest to check that error is nil here and in other context funcs.

Reply:
Same for sender

Reply:
done